tools/generate-rpc-types.ts

```diff
@@ -1,6 +1,3 @@
-// Copyright 2017-2020 @polkadot/apps-config authors & contributors
-// SPDX-License-Identifier: Apache-2.0
-
```

Comment:
@joelamouche I'm assuming this is just a copied comment that can be safely removed. Is that right? I thought you wrote this file. If this copyright should be retained, I can restore it.

Reply:
yup ^^